Source/Painting/SvgPaintServerFactory.cs

```diff
@@ -18,92 +16,44 @@ static SvgPaintServerFactory()
 
         public static SvgPaintServer Create(string value, SvgDocument document)
         {
+            if (value == null)
+                return SvgPaintServer.NotSet;
+
+            var colorValue = value.Trim();
             // If it's pointing to a paint server
-            if (string.IsNullOrEmpty(value))
-            {
+            if (string.IsNullOrEmpty(colorValue))
                 return SvgPaintServer.NotSet;
-            }
-            else if (value == "inherit")
-            {
+            else if (string.Equals(colorValue, "none", StringComparison.OrdinalIgnoreCase))

```

Comment:
I would prefer to use the instance methods here, e.g. `colorValue.Equals("none", StringComparison.OrdinalIgnoreCase)`, though that is a matter of taste.